components/style/properties/longhand/box.mako.rs

```diff
@@ -640,15 +632,49 @@ ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
     use values::NoViewportPercentage;
 
     pub mod computed_value {
-        use std::fmt;
         use Atom;
+        use parser::Parse;
+        use std::fmt;
+        use std::ops::Deref;
         use style_traits::ToCss;
 
-        pub use Atom as SingleComputedValue;
+        #[derive(Clone, Debug, Hash, Eq, PartialEq)]
+        #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+        pub struct AnimationName(pub Atom);
+
+        impl fmt::Display for AnimationName {
+            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+                self.0.fmt(f)
+            }
+        }
+
+        impl Deref for AnimationName {
```

Comment:
In general try to use deref for things that involve actual dereferencing. Just refactor everything else to use `.0` here